styleguide/source/_patterns/02-molecules/jama.json

```diff
@@ -0,0 +1,30 @@
+{
+  "jama": {
+    "links": [
+      {
+        "title": "Surgeon general nominee fought HIV outbreak in rural Indiana",
+        "href": "https://www.google.com",
+        "text": "Surgeon general nominee fought HIV outbreak in rural Indiana",
+        "class": "ama__link-hover-red"
```

Comment:
Should `red` be a modifier here? (e.g. `ama__link-hover--red`)? Also this class doesn't seem to be doing anything, should we keep it here anyway?

Reply:
@labbydev Good point, thanks, I'll change it to an `include` or `extend` or similar. 

Do you think a better name would be `.ama__link--hover-red`? I didn't want to name it `.ama__link--red` because it's not red by default.

Reply:
Yeah I like `.ama__link--hover-red` I think that's probably the most accurate.